docs/best-practices/data-import.rst

```diff
@@ -66,23 +70,24 @@ configuration will be used:
 :replicas:
   1
 
-We recommend you choose the number of shards wisely. They depend on the
-number of nodes in the cluster as well as on the amount of data that goes into
-the table.
+We recommend you choose the number of shards wisely. It depends on the number
+of nodes in the cluster as well as on the amount of data that goes into the
+table.
 
 .. CAUTION::
 
   Be careful, you cannot change the number of shards once the table is created!
 
-Assuming there are 6 nodes in the cluster, and we put 2 shards on each node
-giving us a total of 12 shards, which should be enough for millions of records.
+Assuming there are 6 nodes in the cluster, we put 2 shards on each node,
```

Comment:
"six" instead of "6" per https://oreillymedia.github.io/production-resources/styleguide/#dates_and_numbers

same with "2"

Reply:
"unless the same object appears in a sentence with an object 10 or over": in this case "12 shards". Otherwise you end up with "six nodes, two shards, 12 shards", which looks odd.

Reply:
ah good catch thx